library/billing.inc

```diff
@@ -216,9 +214,6 @@ function updateClaim($newversion, $patient_id, $encounter_id, $payer_id=-1, $pay
           "pid = '$patient_id' AND encounter = '$encounter_id'");
       }
     }
```

Comment:
This seems fine. The only thing that it appears a 0 will do is not show the billing widget on the patient summary screen. And if want that option, then that should be in the standard globals gui:
https://github.com/openemr/openemr/pull/187/commits/8edb6144fa3124b8b30ca51dea7a58615c6ff8b6